app/boards/shields/lily58/lily58_left.conf

```diff
@@ -1,2 +1,6 @@
+# Uncomment to enable encoder
```

Comment:
This is good to have if you're building from a checkout, for users that use the "ZMK Config" user repository approach, they need this in the unified `lily58.conf` file. I'm fine leaving this here, but can you also add it w/ the comment to that file as well?

Thanks!

Reply:
Lines have been added to the unified conf file.